culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+> If you want to go fast, go alone. If you want to go far, go together.
+
+As a community, we value going far over going fast. To go far, we need to stay
+aligned in our values, goals, and practices. One of the ways that we do this is
+through cultivating review culture.
+
+# On Review Culture
+
+Review culture came to us from the practices of software engineers. Large
+codebases are incredibly complex: they contain the ideas and work of hundreds
+of people, manifested in millions of lines of code. They need to remain
+logically consistent, even as they grow and change. A big part of how engineers
+manage is through code review: every single change to the system is reviewed by
+at least one other person before becoming official, or "merging".
+
+Ensuring that all work is reviewed brings several benefits. Most obviously, it
+leads to higher quality work, because the final output includes feedback and
+suggestions from others. Just as importantly, reviews enable the exchange of
+knowledge, context, and perspective within the team. In a good review, both the
+contributor and the reviewer can teach each other about how work gets done on
+the project. Finally, review ensures that every part of the project is
+understood by at least two people.
```

Comment:
Probably the number of reviewers required for merge would vary based on the repository? Later we should figure out the standard for deciding this. Intuitively, small trivial things == one review, if it takes more than 5 minutes, 2 reviewers, and more robust changes (code or docs) should have 3-4.

Reply:
I agree that it depends on project and context, but in my experience it
has been far more effective to bias toward fewer reviewers than more.
Pulling in two reviewers for a non-trivial change is a huge productivity
hit, both because the extra reviewer takes the time to review the code
and because the presence of more people encourages bikeshedding and
nitpicking. In fact, all projects that I’ve been on that started out
with a many-reviewer culture eventually adopted a guideline of “prefer
one reviewer; if you request multiple, explicitly call out what you want
from each person and why one reviewer doesn’t suffice”, or similar.

Reply:
If you've experienced this multiple times, I'm all for fewer reviewers - this feels a bit like (what are they called, anti-patterns?) because intuitively you'd think more is better, but... (insert metaphor with cooks and soup broth). I like the idea of erring toward fewer, and leaving it up to the requester (and possibly the first reviewer) to decide if more are appropriate.

I wasn't familiar with the term bikeshedding - [here is a helpful link](https://css-tricks.com/what-is-bikeshedding/) for others that might not be.